### PR TITLE
Ask to remove boost-1.67 in install script

### PR DESCRIPTION
**Pull request by Bensk1**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 
 if [[ -z $OPOSSUM_HEADLESS_SETUP ]]; then
+    BOOST_INSTALLED=$(dpkg-query -W --showformat='${Status}\n' libboost1.67-dev|grep "install ok installed")
```
Comment (Bensk1) [inline install.sh]: If boost is not found, there is still visible output of dpkg-query

File: install.sh [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+    if [ "" != "$BOOST_INSTALLED" ]; then
+         read -p 'libbost1.67-dev is installed. Deinstall to avoid errors? [y|n] ' -n 1 -r < /dev/tty
```
Comment (mrks) [inline install.sh]: libbost1.67-dev is installed, 1.70 is required. Ok to remove 1.76-dev?

File: install.sh (hunk 1 above)
Comment (mrks) [reply install.sh]: 2>/dev/null

File: install.sh [hunk 3]
```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 
 if [[ -z $OPOSSUM_HEADLESS_SETUP ]]; then
+    BOOST_INSTALLED=$(dpkg-query -W --showformat='${Status}\n' libboost1.67-dev 2>/dev/null | grep "install ok installed")
+    if [ "" != "$BOOST_INSTALLED" ]; then
+         read -p 'libbost1.67-dev is installed, 1.70 is required. Ok to remove 1.76-dev? [y|n] ' -n 1 -r < /dev/tty
```
Comment (mrks) [inline install.sh]: 67, not 76

File: install.sh (hunk 2 above)
Comment (Bensk1) [reply install.sh]: copy pasta

File: install.sh [hunk 4]
```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 
 if [[ -z $OPOSSUM_HEADLESS_SETUP ]]; then
+    BOOST_INSTALLED=$(dpkg-query -W --showformat='${Status}\n' libboost1.67-dev 2>/dev/null | grep "install ok installed")
+    if [ "" != "$BOOST_INSTALLED" ]; then
+         read -p 'libbost1.67-dev is installed, 1.70 is required. Ok to remove 1.67-dev? [y|n] ' -n 1 -r < /dev/tty
```
Comment (tobodner) [inline install.sh]: Spelling error: lib boost

Comment (tobodner) [inline install.sh]: Make aware of conflict: 'X is installed but Y is required'

Comment (tobodner) [inline install.sh]: Align with phrasing below: Remove X?

File: install.sh [hunk 5]
```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 
 if [[ -z $OPOSSUM_HEADLESS_SETUP ]]; then
+    BOOST_INSTALLED=$(dpkg-query -W --showformat='${Status}\n' libboost1.67-dev 2>/dev/null | grep "install ok installed")
```
Comment (tobodner) [inline install.sh]: Maybe add a comment that this is now the place in our install script where we may add necessary upfront cleanup steps.

Comment (mrks) [reply install.sh]: Hopefully this won't become a common pattern

File: install.sh (hunk 4 above)
Comment (mrks) [reply install.sh]: ?

Comment (Bensk1) [reply install.sh]: Below it says `Continue?`

File: install.sh (hunk 5 above)
Comment (tobodner) [reply install.sh]: > Hopefully this won't become a common pattern

Agreed. Anyhow it saves people time to know how to deal with such problems.

**Pull request merged**